javascript/javascript3/week1/weatherApp.js

```diff
@@ -0,0 +1,114 @@
+fetch("https://api.openweathermap.org/data/2.5/weather?q=copenhagen&units=metric&appid=ae80d9c115b082960ef280331a3b25f9")
+    .then(response => response.json())
+    .then(result => console.log(result));
+
+/************************************* */
+
+const inputCity = document.getElementById("inputCity")
+const weatherBtn = document.getElementById("weatherBtn")
+const temp = document.getElementById("temp")
+const icon = document.getElementById("icon")
+const clouds = document.getElementById("clouds")
+const sunrise = document.getElementById("sunrise")
+const sunset = document.getElementById("sunset")
+const map = document.getElementById("map")
+const currentLctBtn = document.getElementById("currentLctBtn")
+const loading = document.getElementById("loading")
+const container = document.getElementById("container")
+const infoBox = document.getElementById("infoBox")
+const message = document.getElementById("message")
+
+//Get City Weather
+weatherBtn.addEventListener("click", getCityWeather)
+function getCityWeather() {
+    let cityName = inputCity.value;
+    infoBox.style.visibility = "visible";
+    fetch(`https://api.openweathermap.org/data/2.5/weather?q=${cityName}&units=metric&appid=ae80d9c115b082960ef280331a3b25f9`)
+        .then(response => response.json())
+        .then(weatherData => {
+            renderWeatherStatus(weatherData)
+            localStorage.setItem("userLocation", renderWeatherStatus(weatherData))
+
+        })
+}
+//Get weather from  user location
+currentLctBtn.addEventListener('click', userLocation);
+function userLocation() {
```

Comment:
Make sure the name of the function always contains atleast one verb

Reply:
Have changed it now and will careful the next time